### DISCUSSION TITLE
Arguments in Bindings definition

### DISCUSSION THREAD

**Discussion by Shahzad24** [rabbitmq/messaging-topology-operator#489] [Q&A] [answered]
Hi, I have use case where an Argument "x-message-ttl : 86400000" needs to be set as part of the binding definition.  (Please see attached file). I can do that using RabbitMQ UI. How do I achieve this using topology operator ?

---
apiVersion: rabbitmq.com/v1beta1
kind: Binding
metadata:
  name: rmqtest-NewQueuesTest
spec:
  vhost: "/"
  source: arc
  destination: NewQueuesTest
  destinationType: queue
  routingKey: NewQueuesTest
  Arguments: ?????
  rabbitmqClusterReference:
    name: rmqtest
![binding](https://user-images.githubusercontent.com/20604085/200135818-658aefe2-559b-486a-ad01-e8521bc76e56.png)

**Comment by ChunyiLyu** [comment, marked as answer]
You can provide extra arguments to bindings in `spec.arguments`.

It could look like:
```
apiVersion: rabbitmq.com/v1beta1
kind: Binding
metadata:
name: test
spec:
vhost: "/"
source: a-source
destination: a-dest
arguments:
  message-ttl: 86400000
```
